core/sphinx/packet_factory/packet_factory.go

```diff
@@ -0,0 +1,136 @@
+package packet_factory
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/katzenpost/katzenpost/core/crypto/rand"
+	"github.com/katzenpost/katzenpost/core/epochtime"
+	"github.com/katzenpost/katzenpost/core/pki"
+	"github.com/katzenpost/katzenpost/core/sphinx"
+
+	"github.com/katzenpost/katzenpost/core/sphinx/constants"
+	"github.com/katzenpost/katzenpost/core/sphinx/geo"
+	"github.com/katzenpost/katzenpost/core/sphinx/path"
```

Comment:
can we import this as `sphinx_path` instead? it reads very generic when you just see `path.PathFactory`.
Like perhaps in *this* module it's fine but in other modules it becomes confusing, I think.

Reply:
fixed